### ISSUE TITLE
applying plan gives error saying bug with terraform

### ISSUE THREAD

**Issue by zaheerm** [hashicorp/terraform#15628]
# PROVIDER ISSUES

**PLEASE NOTE:** Terraform has split out the builtin Providers into their own repositories. For any Provider issues, please open all issues and pull requests in the corresponding repository. An index of supported Providers can be found here:

- https://github.com/terraform-providers

All other issues (that appear to affect multiple or all providers) may be an issue with Terraform's core, and should be opened here. 

------

Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version

0.9.5

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

```variable "aws_access_key" {}
variable "aws_secret_key" {}
variable "aws_region" {
  default = "us-east-1"
}
variable "label" {
  default = "training"
}

provider "aws" {
  access_key = "${var.aws_access_key}"
  secret_key = "${var.aws_secret_key}"
  region = "${var.aws_region}"
}

resource "aws_instance" "web" {
  # ...
  ami                    = "ami-79634a19"
  instance_type          = "t2.micro"
  subnet_id              = "subnet-1e01017a"
  vpc_security_group_ids = ["sg-a3718ec5"]

  tags {
    Identity = "NWI-0724172-tf-deer"
    zme_blah = "hassan"
    zme_blahblah = "hassan"
    Name = "${var.label}"
  }
  count = "2"
}

output "public_ip" {
  value = "${aws_instance.web.public_ip}"
}

output "public_dns" {
  value = "${aws_instance.web.public_dns}"
}
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.
https://gist.github.com/zaheerm/7fa35d0142d50442e0659dc7d28cc701 is the output of terraform apply.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?

Success reported

### Actual Behavior
What actually happened?

The output in https://gist.github.com/zaheerm/7fa35d0142d50442e0659dc7d28cc701 appears

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:

**Comment by jbardin** [comment]
Hi @zaheerm,

Thanks for filing the issue. I have a feeling that this problem has been solved, but there's not enough information here to reproduce the issue. Can you show what steps were taken before `terraform apply`, or the previous configuration or state that was running?

**Comment by zaheerm** [comment]
@jbardin I did this in part of the terraform training that was given by Patrick Salisbury from nebulaworks. The changes that were made to the .tf was the addition of the 2 output clauses.

**Comment by jbardin** [comment]
@zaheerm,

Thanks for the info. The diff error looks like it has to do with the tags, but only on the first host which is strange. The outputs you have are also going to generate errors (though they will only be logged), because you have multiple instances and have to address them separately.

The diff error appears to related to issue fixed up in the releases since 0.9.5, so I'm going to tentatively close this. If you have a standalone reproduction that can demonstrate the error, please feel free to post it and we can reopen the issue.
